include/librealsense2/h/rs_option.h

```diff
@@ -340,6 +340,31 @@ extern "C" {
     */
     const char* rs2_get_option_value_description(const rs2_options* options, rs2_option option, float value, rs2_error ** error);
 
+    /**
+    * Sets a callback in case an option in this options container value is updated
+    * Will create a thread that will periodically check the options in the container for updates
+    * \param[in] options    the options container
+    * \param[in] callback   callback function pointer to update on value changes
+    * \param[out] error     if non-null, receives any error that occurs during this call, otherwise, errors are ignored
+    * \return a subscription to options value changed notifications.
+    * Caller must keep the subscription until it is no longer needed, then canel it using cancel_sucscription().
+    */
+    rs2_subscription * rs2_set_options_changed_callback( const rs2_options * options,
```

Comment:
This is not what I meant about a subscription... let's talk

Reply:
Removed. Subscription is part of `sensor`